go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	github.com/spf13/cobra v1.6.0
 	github.com/vishvananda/netlink v1.1.0
 	go.opentelemetry.io/otel/trace v1.10.0
-	golang.org/x/sys v0.3.0
```

Comment:
qq: in the commit message, it does not mention changes to x/sys. Did you by any chance do something like:
```
❯ go get -u golang.org/x/sys
go: downloading golang.org/x/sys v0.7.0
go: upgraded golang.org/x/sys v0.3.0 => v0.7.0
```
?